.changeset/soft-penguins-perform.md

```diff
@@ -0,0 +1,7 @@
+---
+"@makeswift/runtime": patch
+---
+
+Refactors checkbox control tests to be stricter by asserting against exact DOM
+node content matching for prop values. Also uses data-driven unit tests for more
+exhaustive coverage.
```

Comment:
I think we don't need a changeset if we are not changing any user-visible behavior.

Reply:
Addressed, removing